README.md

```diff
@@ -36,6 +36,16 @@ Example [function](./example_function_test.go) and
 They both fail. Making them pass is a good way to get first real experience
 of working with rapid.
 
+## Usage
+
+Just run `go test` as usual, it will pick up also all `rapid` tests.
+
+There are a number of optional flags to influence rapid behavior, see file `engine.go` or run `go test -args -h` and look at the flags with the `-rapid.` prefix. You can then pass such flags as usual. For example:
```

Comment:
Can you please remove the reference to `engine.go`? This is an implementation detail that might change in the future.

Reply:
Removed reference to engine.go and reformatted to 80 chars.